serving/using-an-ssl-cert.md

```diff
@@ -73,11 +73,16 @@ spec:
 Once the change has been made, you can now use the HTTPS protocol to access
 your deployed services.
 
+## Obtaining an SSL/TLS certificate using Let’s Encrypt through CertBot
 
-## Obtaining an SSL/TLS certificate using LetsEncrypt through CertBot
+If you don't have an existing SSL/TLS certificate, you can use [Let's
+Encrypt][le] to obtain a certificate manually.
 
-If you don't have an existing SSL/TLS certificate, you can use [LetsEncrypt](https://letsencrypt.org)
-to obtain a certificate manually.
+> **Warning:** Certificates issued by [Let's Encrypt][le] are only valid for
+> [90 days](https://letsencrypt.org/docs/faq/). You must renew your certificate
+> with certbot tool again.
```

Comment:
"You must renew your certificate with certbot tool again." --> "You must renew your certificate with the certbot tool every 90 days."